sections/Dataset.tex

```diff
@@ -8,4 +8,10 @@ \section{Dataset}
 
 Currently we extract only the ID, body text, and headline from the xml files, though they do contain more information, such as authors, references to images, article type, date and time of publishing, etc.
 This information could become relevant with the use of various \gls{lda} modifications.
-\vejleder[inline]{make example of an article}
+An example of an article could be a weather report like this one.
+\begin{quote}	
```

Comment:
Vi snakker lige før om at vi trækker ID, body og headline ud, og at der er mere information, men viser kun headline og body.

Reply:
Jeg tænkte at vise ID'et var lidt for meget fordi det er bare et unikt id

Reply:
Følte bare at det er lidt misvisende, da det ikke er et fuldt eksempel, men :shrug: